content/blogposts/2022/cloud-managed-instances-for-enterprise.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Secure, easy, and scalable Sourcegraph Cloud managed instances for the enterprise"
+publishDate: 
+description: TODO
+tags: [blog]
+slug: "enterprise-cloud"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+> NOTE: This post is very WIP.
```

Comment:
@sqs here's the outline of @beyang's blog we discussed late last week: https://docs.google.com/document/d/1dZxT9iiUfnNUw6EZHbhJS0uxdfV53qNV4EMAfjlL6PM/edit#heading=h.ndoqaqxyl2nz